x-pack/lib/filters/geoip/download_manager.rb

```diff
@@ -78,17 +78,24 @@ def download_database(server_db)
     end
   end
 
-  # extract COPYRIGHT.txt, LICENSE.txt and GeoLite2-{ASN,City}.mmdb from .tgz to temp directory
+  # extract all files and folders from .tgz to vendor directory
+  # existing files folders will be replaced
   def unzip(zip_path)
     new_database_path = zip_path[0...-(GZ_EXT.length)] + DB_EXT
     temp_dir = Stud::Temporary.pathname
 
     LogStash::Util::Tar.extract(zip_path, temp_dir)
     logger.debug("extract database to ", :path => temp_dir)
 
+    ::Dir.children(temp_dir).each do |path|
```

Comment:
`path` in this case is a filename, not a full FS path, so probably worth it name `filename`